src/openrct2/ride/ride.c

```diff
@@ -4779,22 +4779,18 @@ static void ride_create_vehicles_find_first_block(rct_ride *ride, rct_xy_element
             break;
         case TRACK_ELEM_DIAG_25_DEG_UP_TO_FLAT:
         case TRACK_ELEM_DIAG_60_DEG_UP_TO_FLAT:
-        case TRACK_ELEM_DIAG_25_DEG_UP:
-        case TRACK_ELEM_DIAG_60_DEG_UP:
```

Comment:
These two are removed, but the do not re-appear anywhere in this switch. Is that correct? I'm not familiar with this piece of code, but perhaps it could use a comment if it is not meant to be part of this switch?